js/components/contactList/contactList_controller.js

```diff
@@ -115,21 +130,19 @@ angular.module('contactsApp')
 	$scope.$watch('ctrl.routeParams.gid', function() {
 		// we might have to wait until ng-repeat filled the contactList
 		ctrl.contactList = [];
-		// watch for next contactList update
-		var unbindWatch = $scope.$watch('ctrl.contactList', function() {
-			if(ctrl.contactList && ctrl.contactList.length > 0 && $(window).width() > 768) {
-				$route.updateParams({
-					gid: $routeParams.gid,
-					uid: ctrl.contactList[0].uid()
-				});
-			} else {
-				$route.updateParams({
-					gid: $routeParams.gid,
-					uid: undefined
-				});
-			}
```

Comment:
Because it was causing issues. It was replaced by this test: https://github.com/owncloud/contacts/blob/dabce47b59347b0b0e81be8f266803d77e4afb62/js/components/contactList/contactList_controller.js#L50